environment.yml

```diff
@@ -15,6 +15,9 @@ dependencies:
 - pylint
 - coverage
 - "paramtools>=0.14.2"
+- curl
+- pandoc
 - pip
 - pip:
     - jupyter-book
+    - pybtex
```

Comment:
@chusloj Is it possible to only install these packages for the "Update Tax-Calculator Use Cases" action?

Reply:
@hdoupe Sure thing.